### DISCUSSION TITLE
请问中文关键词里面的英文字体怎么改成新罗马？

### DISCUSSION THREAD

**Discussion by hulaaa128** [note286/xduts#192] [Q&A] [answered]
原文是黑体

**Comment by note286** [comment]
本科还是研究生？

**Reply by hulaaa128** [reply]
本科，我直接在pdf上面改了！顺便问一下，本科毕设目录圆点怎么改密集，老师说密集一点美观orz

**Reply by note286** [reply, marked as answer]
目前《西安电子科技大学本科毕业设计（论文）撰写规范》里面很多规范要求就是瞎搞，有空我得和教务处沟通下。

---

你可以在`\begin{document}`上面增加一行`\renewcommand{\cftdotsep}{2.5}`，默认值是`4.5`。

**Reply by note286** [reply]
你可以暂时这样解决：

```latex
\xdusetup{info/keywords={\CJKfamily-{rm}测试Test,测试Test,Test,测试}}
```

**Reply by hulaaa128** [reply]
谢谢你！！！！！感恩！！！
